### DISCUSSION TITLE
Explanation of biochemicalProperty export options

### DISCUSSION THREAD

**Discussion by vincentwalter** [milaboratory/mixcr#1851] [Q&A] [answered]
I habe a question about your export options. Your documentation states:

`-biochemicalProperty <gene_feature> <property>`
> Biochemical property of specified gene feature normalized by AA sequence size. Possible values: Hydropathy, Charge, Polarity, Volume,     Strength, MjEnergy, Kf1, Kf2, Kf3, Kf4, Kf5, Kf6, Kf7, Kf8, Kf9, Kf10, Rim, Surface, Turn, Alpha, Beta, Core, Disorder, N2Strength, N2Hydrophobicity, N2Volume, N2Surface

`-baseBiochemicalProperties <gene_feature>`
> Base biochemical properties of specified gene feature normalized by AA sequence size: N2Strength, N2Hydrophobicity, N2Surface, N2Volume, Charge

Is `-baseBiochemicalProperties <gene_feature>` just shorthand for
`-biochemicalProperty <gene_feature> N2Strength -biochemicalProperty <gene_feature> N2Hydrophobicity -biochemicalProperty <gene_feature> N2Surface -biochemicalProperty <gene_feature> Charge`?

Also, how do the N2 versions differ? E.g. what is N2Volume vs Volume?

**Comment by mizraelson** [comment, marked as answer]
Hi,
N2 is a normalized version of a metric. To simplify, we multiplied each metric by a certain coefficient so that they fall approximately within the same range of values, making them easier to present on a plot.

Yes, that is correct, `-baseBiochemicalProperties <gene_feature>` returns a set of metrics that can alternatively be added one by one.

**Reply by vincentwalter** [reply]
Hi Mark, 
Thank you for your reply!
Is this documented somewhere, so I can get a better understanding of it?

**Reply by mizraelson** [reply]
Unfortunately, we do not have a detailed explanation available. You can find references for the metrics here: [MiXCR Post-Analysis CDR3 Metrics](https://mixcr.com/mixcr/reference/mixcr-postanalysis/#cdr3-metrics).

Essentially, there is a table with values for each amino acid (AA) for each metric, and we calculate the sum of these values in the central part of the CDR3.
